### PR TITLE
Introduce (large) CSV import in e2e tests

### PR DESCRIPTION
**Pull request by jaegeral**
Introducing csv upload end2end tests

they do not trigger 
https://github.com/google/timesketch/issues/2796. Fix for that and e2e tests will come with a different PR 
Because the file was not touched a for a while, it also contains some formatting.

### CODE REVIEW

File: importer_client/python/timesketch_import_client/importer.py [hunk 1]
```diff
@@ -723,8 +715,14 @@ def close(self):
...
-        if self._data_lines:
-            self.flush(end_stream=True)
+        # TODO(jaegeral): End of 11/2023 from now remove the code below
+        # if no new bugs occur
+        # Commenting out because of https://github.com/google/timesketch/issues/2796
+        # testing it didn't seem to cause any issues as if flush is called with
+        # end_stream=True it will just return so there is no reason
+        # to call it here.
+        # if self._data_lines:
+        #    self.flush(end_stream=True)
```
Comment (berggren) [inline importer_client/python/timesketch_import_client/importer.py:725]: Please add a test for this. Is this replicable with a test file of > 50k entries (the flush threashold)?

**Review by berggren — CHANGES_REQUESTED** [review]

File: importer_client/python/timesketch_import_client/importer.py (hunk 1 above)
Comment (jaegeral) [reply importer_client/python/timesketch_import_client/importer.py]: removed it for now, I am introducing a test for < 50 k events now, if that is merged in, I will add another for < 50 k events

**Comment by berggren** [comment]
Fix the e2e test

**Comment by jaegeral** [comment]
All green

File: importer_client/python/timesketch_import_client/importer.py [hunk 2]
```diff
@@ -201,7 +201,8 @@ def _fix_data_frame(self, data_frame):
                 data_frame["datetime"] = date.dt.strftime("%Y-%m-%dT%H:%M:%S%z")
             except Exception:  # pylint: disable=broad-except
                 logger.error(
-                    "Unable to change datetime, is it badly formed?", exc_info=True
+                    "Unable to change datetime, is it badly formed?",
```
Comment (berggren) [inline importer_client/python/timesketch_import_client/importer.py:204]:
```suggestion
                    "Unable to change datetime, is it correctly formatted?",
```

**Review by berggren — APPROVED** [review]

**Pull request merged**